orc8r/cloud/go/services/orchestrator/servicers/builder_servicer.go

```diff
@@ -118,7 +120,11 @@ func getMagmadMconfig(
 	if err != nil {
 		return nil, err
 	}
-
+	orc8rVersion, res := os.LookupEnv("VERSION_TAG")
+	if !res {
+		glog.V(1).Info("Couldn't get value for VERSION_TAG Env variable.")
```

Comment:
This should be an error log. Missing this log message could lead to subtle bugs if unaddressed, depending on how we set up the AGW behavior

Reply:
there is a part 2 to this change, where we will change AGW to show a warning when the version differs

Reply:
To me an error log means "something is wrong, but we'll keep going as best we can". So the question is: do we expect VERSION_TAG to always be present? If yes, it's an error, if no, it's an info

Reply:
thats a good question. Do we? For me error is serious business, requiring attention before something catastrophic happens, or first thing i look for when something catastrophic happens.

if we were to stop AGW from running in case the versions differ by more than 1, then it would be important enough to be present, and if its not present, then log error. right now we only print warning on AGW, and with the latest suggestion of making orc8r_version option, even that wont be possible, so it doesnt look like this env variable is that important.

Reply:
At AGW we're only going to print a warning.

At Orc8r, I think we should go ahead and assume the tag will be present. In which case, it would be an error if/when it's missing